robottelo.properties.sample

```diff
@@ -388,6 +388,16 @@ port_range=9091, 14999
 # default value is private
 # managed_ip=Private
 
+#[gce]
+# Google Provider as Compute Resource
+# Project name in Google provider
+#project_id=sat6qe
+# Service Account email id which has compute admin permission
+#client_email=
+# client json Certificate path which is local path on satellite
+#cert_path=
+# Zones
+#zone=
 
```

Comment:
to add support for above , requires to raise a PR at `robottelo-ci`  as well.

Reply:
Raised PR in gitlab PR number 231.